app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/address/enter_postcode.scala.html

```diff
@@ -38,7 +38,7 @@
     backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
     accountMenu: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.account_menu,
     returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
-    caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption
+    caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption_govuk
```

Comment:
I think, we need to use different approach. you can fix it using PageHeading component like 
`label = pageHeading(title, Some(messages("returns.disposal-details.caption"))),`

try this one if works okay. otherwise there is another approach too.